chaos-monkey-docs/src/main/swagger-ui/oauth2-redirect.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
```

Comment:
Is this file generated?

Reply:
Everything in this commit comes from Swagger-UI, i.e. https://github.com/swagger-api/swagger-ui/tree/master/dist. I separated the commits so that it's easier to see what I wrote and what's just copied in.

Swagger-UI is a tool which presents OpenAPI files in an appealing format (i.e., in a webpage).

Reply:
Yeah I know swagger, but I wasnt sure about the oauth2 stuff :D